src/python/dxpy/scripts/dx.py

```diff
@@ -2749,8 +2749,28 @@ def render_timestamp(epochSeconds):
 
 def list_database_files(args):
     try:
+        # check if database was given as an object hash id
+        if is_hashid(args.database):
+            desc = dxpy.api.app_describe(args.database)
```

Comment:
We perform `app_describe` in order to get the class type in the case that the given ID wasn't a database object.

For example:
```
root@localhost:/dx-toolkit# dx list database files file-G594y580KY1BzG478y9KJJxy
Error: The given object is of class file but an object of class database was expected
```

Reply:
app_describe should be used actually for apps https://github.com/dnanexus/dx-toolkit/blob/5fc65a7f452c2b56475d8566824e341cbc621173/src/python/dxpy/api.py#L98. However, it doesn't really matter as all describe methods just invoke id/describe.

Reply:
I think it should be good to use database_describe instead of app_describe even though in the end both are doing same thing, to just make it more readable and not adding any confusion in the future for someone who can get confuse that why app_describe got used for database?? :)

Reply:
Ahh gotcha, that makes sense. I'm going to change this to use `database_describe` instead. Thanks @durae and @ksoni!